stripe/src/main/java/com/stripe/android/view/CountryCode.kt

```diff
@@ -0,0 +1,20 @@
+package com.stripe.android.view
+
+import androidx.annotation.Size
+import java.util.Locale
+
+internal data class CountryCode(
+    @Size(2) private val value: String,
+) {
+    val twoLetters: String
+        get() = value.toUpperCase(Locale.getDefault())
```

Comment:
Why do we need this? The two character code should be locale-independent. We should use `Locale.ROOT`.